src/client/main.tsx

```diff
@@ -3,6 +3,31 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import { installDash } from './components/dash/install';
 
+import sio from 'socket.io-client';
+
+const uri = `${document.location!.origin}`;
+const sioNetwork = sio(uri, {
+  upgrade: false,
+  transports: ['websocket'],
+});
+sioNetwork.on('connect', () => {
+  console.log('Client connected to server');
+});
+sioNetwork.on('reconnect_attempt', () => {
+  console.log('Client reconnecting to server');
+  sioNetwork.io.opts.transports = ['polling', 'websocket'];
+});
+sioNetwork.on('disconnect', () => {
+  console.log('Client disconnected from server');
+});
+sioNetwork.on('udp_packet', (...packet: any[]) => {
```

Comment:
In general, using any is bad, do you know what type the object you get is?

Reply:
The socket.io emit function signature is `emit(event: string, ...args: any[])`. I figured we should keep it the same to allow us the most flexibility in what we send. We could probably be more specific on the type once we have decided what it is we want to send, but its going to get cast to an `any` anyway.